drivers/i2c/i2c_ll_stm32_v1.c

```diff
@@ -647,8 +645,8 @@ static int32_t i2c_stm32_msg_write(const struct device *dev, struct i2c_msg *msg
 
 	i2c_stm32_enable_transfer_interrupts(dev);
 
-	if (k_sem_take(&data->device_sync_sem,
-			K_MSEC(I2C_STM32_TRANSFER_TIMEOUT_MSEC)) != 0) {
+		if (k_sem_take(&data->device_sync_sem,
+			       K_MSEC(CONFIG_I2C_STM32_TRANSFER_TIMEOUT_MSEC)) != 0) {
```

Comment:
Could you restore the original indentation that was fine?
```suggestion
	if (k_sem_take(&data->device_sync_sem,
		       K_MSEC(CONFIG_I2C_STM32_TRANSFER_TIMEOUT_MSEC)) != 0) {
```

Ditto at line 670.